app/controllers/wishlists_controller.rb

```diff
@@ -5,7 +5,7 @@ def show
     skip_authorization
     @wishlist = Wishlist.includes(wishlist_items: :item).find(params[:id])
     @site_managers = @wishlist.users
-    @wishlist_items = @wishlist.wishlist_items
+    @wishlist_items = @wishlist.wishlist_items.ordered
```

Comment:
Looks good! One request: `#ordered` doesn't give any information into _how_ it's ordered. Could you either ditch the scope (i.e. `@wishlist.wishlist_items.order(priority: :desc)`) or change the scope name (e.g. `#priority_order`)? That way, we can see what data to expect at a glance.